prometheus-exporters/ucs-exporter/templates/ucs-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ucs-config
+data:
+  config.yaml: |
+    netbox:
+        netbox_url: "https://netbox.global.cloud.sap"
+        netbox_token: "97bc508ab27df514c963435ae7097672efdaab2a"
+        ucs_hostname_format: "ucscluster-{}.cc.{}.cloud.sap"
+        regions: ["qa-de-1"]
```

Comment:
this needs to be changed to `$values.ucs.netbox_url` etc
make sure in `values.yaml` you set it to `DEFINED-IN-REGION-SECRETS` and then add it to the region secrets.

Reply:
I have updated, please check if the syntax is right.